frontend/src/components/AppHeader.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components/macro'
+
+export default function AppHeader() {
+    return (
+        <Header>
+            <h1>BetterList</h1>
+        </Header>
+    )
+}
+
+const Header = styled.header`
+  background: var(--text-color, beige);
```

Comment:
this var is not included in your `GlobalStyles`
if you intend to use the color beige you may want to remove the `var()` part